docs/packaging.md

```diff
@@ -0,0 +1,53 @@
+# Application Lifecycle
```

Comment:
My concern is not about the content duplication; more about the title itself.  Lifecycle documentation (https://github.com/paypal/squbs/tree/RELEASE-0.7.X/docs) talks about the lifecycle hooks.  This doc is mostly about deployment..  While you can call each as "lifecycle", they are lifecycle in different contexts, IMO..  I would prefer calling this something like "deployment" or something like that..